components/main/GlobalAlert.tsx

```diff
@@ -43,6 +48,28 @@ const GlobalAlert = () => {
   ) : null;
 };
 
+function BookmarkAlert() {
+  return (
+    <div className="nhsuk-grid-row" data-cy="bookmarkAlert">
+      <div className="nhsuk-grid-column-full">
+        <div className="app-global-alert__content">
+          <div className="app-global-alert__message">
+            <h2>We have moved</h2>
+            <p>
+              You are seeing this message because you accessed this site using
+              an old address, we have redirected you here automatically.
+            </p>
+            <p>
+              Please update any bookmarks or password managers to use the new{" "}

```

Comment:
Is that what we do to ensure there's a space before the next bit of text? 😂

Reply:
Apparently so, blame prettier 🤷🏻